{"name": "test_sina_div", "status": "passed", "description": "sina验证:错误的用户名和密码", "start": 1582458406629, "stop": 1582458421097, "uuid": "cb4cd413-e9ba-4869-af74-f7c91994b574", "historyId": "3d7a0ae0893d6d351ec955c19711e2e9", "testCaseId": "7f9ec0c1ed7298b53cc8a89108a1bdcf", "fullName": "test_login.TestShouYe#test_sina_div", "labels": [{"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestShouYe"}, {"name": "host", "value": "PC"}, {"name": "thread", "value": "3920-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}